Compiler.hpp

```diff
@@ -41,13 +41,19 @@ class Compiler {
         TheModule.release();
     }
 
-	Value * emitNegation(Value * value);
-
-	Value * emitAbsolute(Value * value);;
+	Value *emitNegation(Value *value);
```

Comment:
Let's move all these functions below and format all the functions since we already touched this part